tools/validate.py

```diff
@@ -224,10 +224,10 @@ def validate_onnx_model(platform, device_type, model_file, input_file,
     import onnx
     if backend == "tensorflow":
         from onnx_tf.backend import prepare
-        print "valivate on onnx tensorflow backend."
+        print ("valivate on onnx tensorflow backend.")
```

Comment:
Remove the space between `print` `(`.

Reply:
Hi, @llhe 
No problem! I believe the spaces are removed now. This is a nice library and I will most likely contribute more. :)

Reply:
Thanks.